pkg/infra/pulumi_aws/iac/eks.ts

```diff
@@ -53,7 +53,7 @@ export interface EksExecUnit {
 export interface HelmChart {
     Name: string
     Directory: string
-    Values: Value[]
+    Values: Value[] | null
```

Comment:
Do you specifically want `null`, or is `undefined` okay?

If `undefined` is okay, then I think the more typescripty way to do this would be:

```suggestion
    Values?: Value[]
```

Reply:
go embed is rendering it to null, so in this case we would specifically want null. Figured its easier to make a 2 line change here than muck around with the go templating stuff